src/configuration/FirestoreConfigurationContext.tsx

```diff
@@ -398,12 +399,15 @@ function getConfigurableLinksFromSnapshot(
   const ticketingInfo = mapLanguageAndTextType(urls?.get('ticketingInfo'));
   const inspectionInfo = mapLanguageAndTextType(urls?.get('inspectionInfo'));
   const termsInfo = mapLanguageAndTextType(urls?.get('termsInfo'));
+  const refundInfo = mapLanguageAndTextType(urls?.get('refundInfo'));
 
-  if (!ticketingInfo || !inspectionInfo || !termsInfo) return undefined;
+  if (!ticketingInfo || !inspectionInfo || !termsInfo || !refundInfo)
+    return undefined;
```

Comment:
Does this make it necessary for every transport company to have a refund info link?

Reply:
You're absolutely correct. I'll update the PR to fix this. Thanks!